lib/tasks/hyacinth/setup.rake

```diff
@@ -74,70 +74,23 @@ namespace :hyacinth do
 
     desc "Add Descriptive Metadata category and Title dynamic fields"
     task seed_dynamic_field_entries: :environment do
-      user = User.find_by(email: default_user_accounts[0][:email])
-      unless user
-        puts Rainbow("seed_dynamic_field_entries requires default Admin user. Run default_users task first").red.bright
-        next
-      end
-
-      df_category = DynamicFieldCategory.find_by(display_label: "Descriptive Metadata")
-      if df_category
-        puts Rainbow("dynamic field category 'Descriptive Metadata' already exists (skipping).").blue.bright
-      else
-        df_category = DynamicFieldCategory.create!(display_label: "Descriptive Metadata")
-        puts Rainbow("Created dynamic field category 'Descriptive Metadata'").green
-      end
-
-      df_group = DynamicFieldGroup.find_by(string_key: "title")
-      if df_group
-        puts Rainbow("dynamic field group 'Title' already exists (skipping).").blue.bright
-      else
-        df_group = DynamicFieldGroup.create!(
-          string_key: 'title',
-          parent: df_category,
-          display_label: 'Title',
-          created_by: user,
-          updated_by: user,
-          parent_type: "DynamicFieldCategory"
-        )
-        puts Rainbow("Created dynamic field group 'Title'").green
-      end
-
-      default_dynamic_fields = [
-        {
-          display_label: 'Sort Portion',
-          string_key: 'sort_portion',
-          dynamic_field_group: df_group,
-          field_type: 'string',
-          created_by: user,
-          updated_by: user,
-          filter_label: '',
-          controlled_vocabulary: '',
-          select_options: '{}',
-          additional_data_json: '{}'
-        },
-        {
-          display_label: 'Non-Sort Portion',
-          string_key: 'non_sort_portion',
-          dynamic_field_group: df_group,
-          field_type: 'string',
-          created_by: user,
-          updated_by: user,
-          filter_label: '',
-          controlled_vocabulary: '',
-          select_options: '{}',
-          additional_data_json: '{}'
-        }
-      ]
+      field_definitions = {
+        dynamic_field_categories: [{
+          display_label: "Descriptive Metadata",
+          dynamic_field_groups: [
```

Comment:
I added these to my pull request too, haha.

Reply:
Yea, it's crazy how many lines it reduces.